### DISCUSSION TITLE
Hardware button to reset and unpair

### DISCUSSION THREAD

**Discussion by Sill1000** [HomeSpan/HomeSpan#931] [Ideas]
Hello,

Instead of connecting an terminal to reset all or unpair device, why do nut use a pushbutton.

Short press unpair
Long press reset

Thanks

**Comment by frankonski** [comment]
Already built-in. That would be option 4 of getting the device into configuration mode. See https://github.com/HomeSpan/HomeSpan/blob/master/docs/UserGuide.md#device-configuration-mode

when used with `setControlPin()` method.
